locale/lt/LC_MESSAGES/crm.po

```diff
@@ -261,6 +263,171 @@ msgstr ""
+"LAU 1 dalis."
+
+#: ../../crm/customizations/base_address_lau.rst:43
+msgid "Valid rule of dependencies: Country > State > LAU 1 > LAU 2."
+msgstr "Galiojanti priklausomybių taisyklė: Šalis > Regionas > LAU 1 > LAU 2."
+
+#: ../../crm/customizations/base_address_lau.rst:46
+msgid "Setup LAU 1 & LAU 2"
+msgstr "LAU 1 ir LAU 2 nustatymai"
+
+#: ../../crm/customizations/base_address_lau.rst:48
+msgid ""
+"LAU 1 and LAU 2 can be reached and configured via corresponding menus: "
+":menuselection:`Contacts --> Configuration --> Localization --> LAU 1` "
+"and :menuselection:`Contacts --> Configuration --> Localization --> LAU "
+"2`."
+msgstr ""
+"LAU 1 ir LAU 2 konfigūruojami ir pasiekiami per atitinkamus meniu: "
+":menuselection:`Kontaktai --> Nustatymai --> Lokalizacija --> LAU 1` ir "
+":menuselection:`Kontaktai --> Nustatymai --> Lokalizacija --> LAU 2`."
+
+#: ../../crm/customizations/base_address_lau.rst:57
+msgid "LAU 1 & LAU 2 in Address"
+msgstr "LAU 1 ir LAU 2 adrese"
+
+#: ../../crm/customizations/base_address_lau.rst:59
+msgid "Currently, LAU 1 and LAU 2 can be set in address on partner:"
+msgstr "Dabar LAU 1 ir LAU 2 gali būti nurodomi partnerio adrese:"
+
+#: ../../crm/customizations/base_address_lau.rst:64
+msgid "or company:"
+msgstr "arba įmonės adrese:"
+
+#: ../../crm/customizations/base_address_lau.rst:69
+msgid "LAU 1 and LAU 2 can also be used in address format of the country."
+msgstr "Be to, LAU 1 ir LAU 2 gali būti naudojami ir šalies adreso formate."
+
+#: ../../crm/customizations/calendar_address.rst:3
+msgid "More Detailed Address in Calendar Event"
+msgstr "Detalesnis adresas kalendoriaus įvykyje"
+
+#: ../../crm/customizations/calendar_address.rst:5
+msgid "Define more detailed location address for calendar event."
+msgstr "Nurodykite tikslesnį kalendoriaus įvykio vietos adresą."
+
+#: ../../crm/customizations/calendar_address.rst:10
+msgid ""
+"For this feature to work, module :code:`calendar_address` must be "
+"installed."
+msgstr ""
+"Kad šis funkcionalumas veiktų, turi būti įdiegtas "
+":code:`calendar_address` modulis."
+
+#: ../../crm/customizations/calendar_address.rst:14
+msgid "Address of Event"
+msgstr "Įvykio adresas"
+
+#: ../../crm/customizations/calendar_address.rst:16
+msgid ""
+"*Address* section is added to include more detailed information about the"
+" address of event."
+msgstr ""
+"Papildomoje *Adresas* skiltyje galima pridėti daugiau informacijos "
+"apie įvykio adresą."
+
+#: ../../crm/customizations/calendar_address.rst:23
+msgid "Auto Fill"
+msgstr "Automatinis užpildymas"
+
+#: ../../crm/customizations/calendar_address.rst:25
+msgid ""
+"Select a partner address in the field with placeholder *Address* to auto "
+"fill all address fields with data from selected partner. This will also "
+"concatenate all values from address fields and insert it into *Location* "
+"field."
+msgstr ""
+"Pasirinkite partnerio adresą lauke, kur lauko žymena yra *Adresas*, ir visi "
```

Comment:
maybe žyma? Is there such word žymena?

Reply:
I've searched for a better translation of `placeholder` (and 'žymena' seemed the best to me), e.g.
http://www.vlkk.lt/konsultacijos/2241-placeholder-rezervuota-vieta-vietos-zymena
We can also use 'vietaženklis', but it sounds even more weird :D